packages/shared/src/graphql/fragments.ts

```diff
@@ -117,11 +117,15 @@ export const COMMENT_FRAGMENT = gql`
   fragment CommentFragment on Comment {
     id
     contentHtml
+    content
```

Comment:
Why is this addition needed, mainly asking becuase comments are already heavy to load and adding more fields will just make them slower.

If it's just for the edit we actually talked about just doing it from a seperate call on the edit itself.
(but let me know what other placement you needed it for)

Reply:
Yes, it's just for the edit.